projects/plugins/jetpack/tests/php/modules/subscriptions/test_class.jetpack-subscriptions.php

```diff
@@ -352,6 +352,13 @@ public function test_comments_are_not_displaying_on_not_pages() {
 	 * @return void
 	 */
 	public function test_comments_are_displaying_on_not_accessible_pages() {
+		add_filter(
+			'jetpack_active_modules',
```

Comment:
Do we need to do some cleanup to remove the filter after the test, to avoid any kind of pollution this test may leave behind with the filter?

Reply:
Isn't that filter only available under this test?